docs/ABOUT.md

```diff
@@ -1,16 +1,9 @@
-PureScript is a purely functional, statically-typed programming language with
-type inference.
+PureScript is a purely functional, statically-typed programming language with global type inference.
 
-**Functional** means that functions are first-class data types.
+**Functional** means computation is modeled as the evaluation of expressions rather than the execution of a sequence of instructions.
```

Comment:
I would say this is a definition of "declarative" more than "functional", but I'm nitpicking here. Maybe we can define both?